fastlane/lanes/build.rb

```diff
@@ -12,13 +16,56 @@
 
   build_for_app_store_connect
 
+  upload_to_app_store_connect(
+    beta_release: beta_release,
+    skip_prechecks: skip_prechecks,
+    create_release: create_release
+  )
```

Comment:
The lane `build_and_upload_to_app_store_connect` is now unused... I'd say we're better off deleting it. It sounds convenient for local usages, but we shouldn't do local builds unless in the most rare of occasions.

Reply:
Agreed, let's delete it.

In the rare cases where we might need to do an App Store build from our local machines to debug stuff or unblock an automation bug, we can always call `fastlane build_for_app_store_connect` then `fastlane upload_to_app_store_connect` separately manually anway.